.github/workflows/ci.yml

```diff
@@ -103,7 +103,7 @@ jobs:
           OS_NAME: osx
       - name: Archive app
         run: >
-          pushd ~/go/src/github.com/digitalbitbox/bitbox-wallet-app/frontends/qt/build/osx;
+          pushd ~/go/src/github.com/bitboxswiss/bitbox-wallet-app/frontends/qt/build/osx;
```

Comment:
fyi it's BitBoxSwiss (even though both work), I prefer using the capitalized one for consistency.

Reply:
done, I used lowercase as most other stuff in go directory is lowercase, but Microsoft :P